node_modules/col-users/lib/api.js

```diff
@@ -65,7 +65,7 @@ var getUser = module.exports.getUser = function(id, callback) {
       'include': [{
         'model': DB.Canvas,
         'as': 'canvas',
-        'attributes': ['canvas_api_domain', 'use_https']
+        'attributes': ['canvas_api_domain', 'use_https', 'logo', 'name']
```

Comment:
Might be worth creating a `BASIC_CANVAS` property list to use every time a basic canvas instance needs to be retrieved